tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py

```diff
@@ -707,6 +707,7 @@
     caching_rtol=3e-3,
     caching_atol=4e-4,
     always_repeat_timestep=False,
+    enable_fast_compositional=True,
```

Comment:
Just a comment.

I prefer to use non default values for those test inputs when possible.
The "IO" test will probably pass even if the value is not being written/read right.